settings.gradle.kts

```diff
@@ -16,5 +16,8 @@ include("kotlin-reference-server")
 /** Integration tests */
 include("integration-tests")
 
+/** E2E tests */
+include("end-to-end-tests")
```

Comment:
We don't have the `build.gradle` under `end-to-end`. This may be confusing. I think we may add the sub-project when we have some Kotlin source code.

Reply:
It makes sense. I'll remove it for now